mlflow/pyfunc/__init__.py

```diff
@@ -127,13 +136,28 @@ def load_pyfunc(path, run_id=None):
         raise Exception("Format '{format}' not found not in {path}.".format(format=FLAVOR_NAME,
                                                                             path=conf_path))
     conf = model.flavors[FLAVOR_NAME]
+    model_py_version = conf.get(PY_VERSION) if PY_VERSION in conf else None
```

Comment:
`conf.get(PY_VERSION)` returns None if PY_VERSION not in conf, so we don't need the rest of the conditional.

Reply:
Fixed!